bcycle-backend/Models/Entities/GroupTrip.cs

```diff
@@ -52,15 +49,12 @@ public void Update(GroupTripRequest data)
                 TripCode = TripCode,
                 StartDate = StartDate,
                 EndDate = EndDate,
-                SharingUrl = GetSharingUrl(urlBase, groupTripSharePrefix),
```

Comment:
We can remove `SharingGuid` from the DB entity as well.

Reply:
Done